### ISSUE TITLE
Container v1: Capsule Acceptance Test Failure

### ISSUE THREAD

**Issue by jtopjian** [gophercloud/gophercloud#1033]
Recently an error has been happening with the `TestCapsule` acceptance test:

```
convenience.go:35: Failure in capsules_test.go, line 84: unexpected error "parsing time "2018-06-13 05:25:24" as "2006-01-02 15:04:05-07:00": cannot parse "" as "-07:00"
```

@kevin-zhaoshuai Do you know if this is a recent change in Zun or is this a temporary error in devstack?

**Comment by jtopjian** [comment]
Look like this is the commit which is causing problems:

https://github.com/openstack/zun/commit/647e44392a26f02283e38de84eb1415b71be0077

We'll need to now support both time formats since this has all been done within a single API version.

**Comment by kevin-zhaoshuai** [comment]
@jtopjian Just find this issue. Will modify the code to support both time formats now.

**Comment by jtopjian** [comment]
@kevin-zhaoshuai Actually I'm working on it right now :)

**Comment by kevin-zhaoshuai** [comment]
@jtopjian That will be great! Thanks a lot

**Comment by jtopjian** [comment]
Fixed in #1042
